README.md

```diff
@@ -30,3 +31,55 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+## Analysis
+
+if statement -> constant... + 1
+
+else statement ->
+
+-for loops $n * n * n * n * n = n^5$
+
+-recursion $3T(n/3)$ -> 3 recursion calls with n/3 each time
+
+=> $3T(n/3)+n^5$
+
+plug it into itself
+
+=> $3(3T(n/3/3)+(n/3)^5) + n^5$     =>     $9T(n/9)+3(n/3)^5 + n^5$
+
+one more time
+
+=> $27T(n/27) + 9(n/9)^5 + 3(n/3)^5 + n^5$ 
+
+=> $3^iT(n/3^i) + \sum_{j=0}^{i-1} 3^j (n/3^j)^5$ 
+
+because constants can be ignored, we can write $\sum_{j=0}^{i-1} 3^j(n/3^j)^5$ as just $n^5$
```

Comment:
Not following you here. The sum isn't a constant.

Reply:
My reasoning here is not that the sum is a constant but that there are constants involved that aren't necessary to the analysis: n^5 + 3(n/3)^5 + 9(n/9)^5

once the sum is completed and the constants are ignored, n^5 is the part that remains

Reply:
Can you then show the solution of the sum first please?